apps/central-scan/backend/src/central_scanner_app.ts

```diff
@@ -7,6 +7,7 @@ import { assert } from '@votingworks/basics';
 import {
   exportCastVoteRecordReportToUsbDrive,
   Exporter,
+  VX_MACHINE_ID,
```

Comment:
Can you do one more check to make sure you caught all the places we use this? Here's one I know of that I don't think you converted: https://github.com/votingworks/vxsuite/blob/drew/cvr-cdf-exports/apps/scan/backend/src/machine_config.ts#L5

Reply:
I left that one because it seemed different and I didn't want to touch it, but will change it now.